media/client/main/source/WebAudioPlayer.cpp

```diff
@@ -43,18 +44,22 @@ std::shared_ptr<IWebAudioPlayerFactory> IWebAudioPlayerFactory::createFactory()
     return factory;
 }
 
-std::unique_ptr<IWebAudioPlayer> WebAudioPlayerFactory::createWebAudioPlayer(std::weak_ptr<IWebAudioPlayerClient> client,
-                                                                             const std::string &audioMimeType,
-                                                                             const uint32_t priority,
-                                                                             const WebAudioConfig *config) const
+std::unique_ptr<IWebAudioPlayer> WebAudioPlayerFactory::createWebAudioPlayer(
+    std::weak_ptr<IWebAudioPlayerClient> client, const std::string &audioMimeType, const uint32_t priority,
+    const WebAudioConfig *config, std::weak_ptr<client::IWebAudioPlayerIpcFactory> webAudioPlayerIpcFactory,
+    std::weak_ptr<client::IClientController> clientController) const
 {
     std::unique_ptr<IWebAudioPlayer> webAudioPlayer;
     try
     {
+        std::shared_ptr<client::IWebAudioPlayerIpcFactory> wapif = webAudioPlayerIpcFactory.lock();
```

Comment:
same here. Please apply to all cases in which you added new object with non explanatory names

Reply:
Done

Reply:
Done